src/android/com/adobe/phonegap/push/IncomingCallActivity.kt

```diff
@@ -0,0 +1,223 @@
+package com.adobe.phonegap.push
+
+import android.Manifest
+import android.annotation.SuppressLint
+import android.app.Activity
+import android.app.KeyguardManager
+import android.content.BroadcastReceiver
+import android.content.Context
+import android.content.Intent
+import android.content.IntentFilter
+import android.content.pm.PackageManager
+import android.graphics.drawable.Drawable
+import android.os.Build
+import android.os.Bundle
+import android.os.Handler
+import android.os.Looper
+import android.util.Log
+import android.view.WindowManager
+import android.widget.Button
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.core.app.ActivityCompat
+import androidx.core.app.NotificationCompat
+import androidx.core.app.NotificationManagerCompat
+import androidx.vectordrawable.graphics.drawable.Animatable2Compat
+import androidx.vectordrawable.graphics.drawable.AnimatedVectorDrawableCompat
+
+private const val POST_NOTIFICATIONS_REQUEST_CODE = 8234
+
+class IncomingCallActivity : Activity() {
+
+    var caller: String = ""
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        showWhenLockedAndTurnScreenOn()
+        super.onCreate(savedInstanceState)
+        Log.d("", "IncomingCallActivity.onCreate()")
+        val activityIncomingCallRes = resources.getIdentifier("activity_incoming_call", "layout", packageName)
+        setContentView(activityIncomingCallRes)
+        window.setFlags(
+            WindowManager.LayoutParams.FLAG_FULLSCREEN,
+            WindowManager.LayoutParams.FLAG_FULLSCREEN
+        )
+        instance = this
+
+        val tvCallerRes = resources.getIdentifier("tvCaller", "id", packageName)
```

Comment:
Check: is it possible for other people to customize this screen?

Reply:
Plz make a screenshot for us to review this call screen to decide what needs to be anonymized